### PR TITLE
grpc-js: Client waitForReady callback fix.  Fixes #1352

### LINKED ISSUE #1352: grpc-js: waitForReady called multiple times
### Problem description
When using `client.waitForReady`, the ready callback is called multiple times (3 times in this case). Is this is the expected behaviour? I expected it to be called only once. Perhaps I'm misunderstanding the API.

### Reproduction steps
 I'm constructing the gRPC client (which extends from [Client.ts](https://github.com/grpc/grpc-node/blob/master/packages/grpc-js/src/client.ts)).

```ts
import * as grpc from '@grpc/grpc-js';
import { GradleTasksClient as GrpcClient } from './java-gradle-tasks-grpc/src/main/proto/gradle_tasks_grpc_pb';

const grpcClient = new GrpcClient(
  'localhost:8887',
  grpc.credentials.createInsecure()
);
const deadline = new Date();
deadline.setSeconds(deadline.getSeconds() + 1);
grpcClient.waitForReady(deadline, (err: Error | undefined) => {
  console.log('is ready');
});
```

### Environment
 - OS name, version and architecture: macos Catalina 10.15.3 x86_64
 - Node version: v12.16.1
 - Node installation method: nvm
 - Package name and version: `@grpc/grpc-js@0.7.9`

### Screencast

grpc-js-waitforready

### PR DESCRIPTION
**Pull request by badsyntax**
Fixes #1352

### CODE REVIEW

**Comment by badsyntax** [comment]
Change the commit messages to follow convention of `grpc-js:`

File: packages/grpc-js/test/test-client.ts [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+  let server: Server;
+  let client: ServiceClient;
+
+  before(done => {
+    const protoFile = path.join(__dirname, 'fixtures', 'echo_service.proto');
+    const echoService = loadProtoFile(protoFile)
+      .EchoService as ServiceClientConstructor;
+
+    server = new Server();
+    server.addService(echoService.service, {
```
Comment (badsyntax) [inline packages/grpc-js/test/test-client.ts:44]: Does this test rely on a service being added to the server? I don't think client.waitForReady depends on this, see https://github.com/grpc/grpc/blob/master/doc/wait-for-ready.md and https://grpc.github.io/grpc/node/grpc.Client.html#waitForReady

Comment (murgatroid99) [reply packages/grpc-js/test/test-client.ts]: No, `waitForReady` should not depend on the server having any services. In fact, the whole test file can be simplified significantly by omitting all of the protobuf stuff and directly using `grpc.Client`.

Comment (badsyntax) [reply packages/grpc-js/test/test-client.ts]: I'll simplify the tests.

Comment (badsyntax) [reply packages/grpc-js/test/test-client.ts]: I've made that change.

**Comment by murgatroid99** [comment]
Thank you for your contribution.

**Pull request merged**